INSTALL.Mac.md

```diff
@@ -261,6 +261,23 @@ Once in the text editor, ensure that the file looks similar to the following:
 </Directory>
 ```
 
+**NOTE FOR OSX10/Apache 2.4 users:** 
```

Comment:
Specify the OSX version. As I know Yosemite has very different Apache install procedures from the rest. 

Other than that looks good :)